### PR TITLE
Feat:Added liked videos page/history/Explore

### PR DESCRIPTION
**Pull request by talrejah02**
Added Explore/liked videos/history Pages
please review and give suggestions.
Use this live [link]()

### CODE REVIEW

File: src/components/videoCard/videoCard.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  
+  return (
+    <div className="card-container">
+      <img className="card-img" src="Images/videoImages/backpack.webp" />
+      <section className="card-description">
+        <section className="card-description-one">
+          <img className="card-logo-img" src="Images/hikeslogo.webp" />
+          <span className="card-title">
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Enim,
```
Comment (Jia-A) [inline src/components/videoCard/videoCard.jsx:15]: I think you forgot to change this Lorem Ipsum.

File: src/components/videoCard/videocard.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
+.card-container {
+    margin: 1rem;
+    display: flex;
+    flex-direction: column;
+    flex-basis: 18rem;
+    background-color: var(--background-secondary);
+    border-radius: 12px;
```
Comment (Megha-Joshi) [inline src/components/videoCard/videocard.css:7]: Convert px->rem

**Review by Jia-A — APPROVED** [review]
UI looks nice.
Your History.css is empty, you might want to look at that. 
Good to merge.

**Review by Megha-Joshi — APPROVED** [review]
- UI looks amazing.
- Nice use of CSS variables.
- Good to merge

File: src/components/videoCard/videoCard.jsx (hunk 1 above)
Comment (talrejah02) [reply src/components/videoCard/videoCard.jsx:15]: its itentional.

File: src/components/videoCard/videocard.css (hunk 2 above)
Comment (talrejah02) [reply src/components/videoCard/videocard.css:7]: sure will do

**Pull request merged**